docs/guides/deployment/overriding-job-variables.md

```diff
@@ -0,0 +1,177 @@
+# Deeper Dive: Overriding Work Pool Job Variables
```

Comment:
thoughts on renaming this to "2 Dive 2 Deep"?

Reply:
😂